pkg/rtc/room.go

```diff
@@ -92,6 +92,19 @@ func (r *Room) GetParticipant(identity string) types.Participant {
 	return r.participants[identity]
 }
 
+func (r *Room) GetParticipantBySid(participantID string) types.Participant {
```

Comment:
Sooner or later, looks like we will need a fast look up of participant in room by either identity or sid. So, maybe we need to maintain two maps.

Reply:
I think a identity to sid map makes a ton of sense. and keying off of sid internally.